### ISSUE TITLE
the model.ckpt generated by the ssd_lite is too big

### ISSUE THREAD

**Issue by 1453042287** [tensorflow/models#5654]
### System information
- **What is the top-level directory of the model you are using**:
object_detection
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Linux Ubuntu 16.04
- **TensorFlow installed from (source or binary)**:
pypi tensorflow-gpu
- **TensorFlow version (use command below)**:
1.11
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**:
CUDA9.0 cuDNN7.3
- **GPU model and memory**:
TITAN V 11G
- **Exact command to reproduce**:
CUDA_VISIBLE_DEVICES=0 python object_detection/model_main.py --pipeline_config_path=project/ssdlite_mobilenet_v2_coco_2018_05_09/ssdlite.config --model_dir=project/ssdlite_result --num_train_steps=100000 --sample_1_of_n_eval_examples=1 --alsologtostderr

### Describe the problem
i use the api and the ssdlite_mobilenet_v2 pre-trained model to fine-tune my own dataset, but the size of the model.ckpt-200000.data-00000-of-00001 generated by the api is huge, almost 39M, it's weird cause the model.ckpt of the ssdlite in the model zoo is only 17M, why?

**Comment by BowieHsu** [comment]
@1453042287 because checkpoints saved both weights and gradient，which should double the size of binary file

**Comment by 1453042287** [comment]
@BowieHsu thanks a lot, but did you mean the ExponentialMovingAverage in the checkpoint is the gradient? or something else, cause i check the name in the checkpoint and i find the ExponentialMovingAverage is the primary difference

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
